pkgs/development/compilers/blueprint/default.nix

```diff
@@ -1,23 +1,31 @@
-{ fetchFromGitLab
+{ blueprint-compiler
+, dbus
+, diffutils
+, fetchFromGitLab
+, gcc
+, gnat
 , gobject-introspection
 , gtk4
 , lib
+, libadwaita
+, makeFontsConf
 , meson
 , ninja
 , python3
 , stdenv
 , testers
+, xvfb-run
 }:
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation rec {
```

Comment:
Why is this reverted?

Reply:
I did browse through nixpkgs and thought rec is used more often and looked cleaner. Is there any substantial difference between using finalAttrs and rec?

Reply:
`finalAttrs` is the newer and better method.